lib/cap_authors_poller.rb

```diff
@@ -126,6 +126,11 @@ def queue_author_for_harvest(author, skip_message)
   end
 
   def update_existing_contributions(author, incoming_authorships)
+    # inject author identity into the authorship data
+    incoming_authorships = incoming_authorships.dup.map do |contrib|
+      contrib[:cap_profile_id] = author.cap_profile_id
+      contrib
+    end
```

Comment:
For more perspective on this, see also helper methods in
https://github.com/sul-dlss/sul_pub/blob/master/app/api/sul_bib/authorship_api.rb